src/services/auth.services.js

```diff
@@ -15,13 +15,11 @@ import{
  * @throws {Error} - If an error occurs while obtaining the installation ID.
  */
 export const getOcktokitClient = async (owner, repo) => {
-
-  const ghApp = new App({
-    appId: GITHUB_APP_IDENTIFIER,
-    privateKey: GITHUB_APP_PRIVATE_KEY,
-  });
-
```

Comment:
I moved ghApp instantiation inside of the try/catch block so that errors instantiating the app would be caught and propagated up the call stack. When I was setting up the app in my local environment and trying to get my private key properly formatted, having the ghApp instantiation outside of the try/catch block made it difficult to debug.

Reply:
But why? Should it not make a big difference? What was the specific problem you had with that?

Reply:
From @BigSamu :

> But why? Should it not make a big difference? What was the specific problem you had with that?

Reply:
I created an issue in our repo to add a separate try/catch block to handle any problems with instantiating `ghApp`.